ansible_collections/arista/avd/roles/eos_designs/docs/tables/connected-endpoints.md

```diff
@@ -319,18 +344,81 @@
                 enabled: <bool>
                 always: <bool>
                 host_mode_common: <bool>
+
+              # Operate interface in per-mac access-list mode.
+              mac_based_access_list: <bool>
               timeout:
                 idle_host: <int; 10-65535>
                 quiet_period: <int; 1-65535>
 
-                # Range 60-4294967295 or "server".
+                # Value can be 60-4294967295 or 'server'.
                 reauth_period: <str>
                 reauth_timeout_ignore: <bool>
                 tx_period: <int; 1-65535>
               reauthorization_request_limit: <int; 1-10>
               unauthorized:
                 access_vlan_membership_egress: <bool>
                 native_vlan_membership_egress: <bool>
+              eapol:
+                disabled: <bool>
+                authentication_failure_fallback_mba:
+                  enabled: <bool>
+                  timeout: <int; 0-65535>
+              aaa:
```

Comment:
maybe lets add these in an eos_designs unit test for completness?

Reply:
There is zero change to the code. We are not testing every possible key in structured config.